Assets/Modules/Pepper2000/P2K_CIV4GameText.xml

```diff
@@ -2079,15 +2084,15 @@
 		<English>Construction Ship</English>
 		<French>Vaisseau de construction
 		</French>
-	</TEXT>
+	</TEXT><!--
```

Comment:
This will cause TXT key issues. Instead write some text

Reply:
fixed, hopefully (will upload fix)

Reply:
TBD texts should be eventually get actual text.
We have 2238 such lines in all languages.

Reply:
I know, I'm trying to do my part, but time is what it is...